README.md

```diff
@@ -46,29 +47,60 @@ use pest::Parser;
 struct IdentParser;
 
 fn main() {
-    let pairs = IdentParser::parse_str(Rule::ident, "abc123").unwrap_or_else(|e| panic!("{}", e));
+    let pairs = IdentParser::parse_str(Rule::ident_list, "abc123 def456").unwrap_or_else(|e| panic!("{}", e));
 
+    //Because ident_list is silent, the iterator will contain idents
     for pair in pairs {
-        match pair.as_rule() {
-            Rule::alpha => println!("letter: {}", pair.into_span().as_str()),
-            Rule::digit => println!("digit: {}", pair.into_span().as_str()),
-            _ => unreachable!() // ident rule is silent and cannot be reached
-        };
+        //A pair is a combination of the rule which matched and a span of input
+        println!("Rule: {:?}", pair.as_rule());
+        println!("Span: {:?}", pair.clone().into_span());
+        println!("Text: {}", pair.clone().into_span().as_str());
+
+        //A pair can be converted to an iterator of the tokens which make it up:
+        for inner_pair in pair.into_inner() {
+            match inner_pair.as_rule() {
+                Rule::alpha => println!("letter: {}", inner_pair.into_span().as_str()),
+                Rule::digit => println!("digit: {}", inner_pair.into_span().as_str()), 
+                _ => unreachable!()
+            };
+        }
     }
 }
 ```
 
+This produces the following output:
+```
+Rule: ident
+Span: Span { start: 0, end: 6 }
+Text: abc123
+letter: a
+letter: b
+letter: c
+digit: 1
+digit: 2
+digit: 3
+Rule: ident
+Span: Span { start: 7, end: 13 }
+Text: def456
+letter: d
+letter: e
+letter: f
+digit: 4
+digit: 5
+digit: 6
+```
+
 ## Meaningful error reporting
 
-Parsing `"123"` instead of `"abc123"` in the code above will result in the following panic:
+Parsing `"123"` instead of `"abc123 def456"` in the code above will result in the following panic:
```

Comment:
This particular example is a little unfortunate because it no longer highlights the "unexpected" error report. Maybe it would be more interesting if only the first ident was not allowed to start with a number? Then we can have one example where we parse `"123"` which errors with "unexpected digit" and one with `"abc *"` which errors with `"expected ident"`.